src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapePlayer.java

```diff
@@ -141,6 +143,26 @@ public boolean canBeArrested() {
         return _isWanted || _inRestrictedArea;
     }
 
+//	########################################
+//	#                Money                 #
+//	########################################
+
+    public int getBalance() {
+        return _balance;
+    }
+
+    public void setBalance(int balance) {
+        _balance = balance;
+    }
+
+    public void increaseBalance(int missionPay) {
+        _balance += missionPay;
+    }
+
+    public void decreaseBalance(int itemPrice) {
```

Comment:
You don't have to be specific in the variable name, you can just name it amount or something vague. If we use this method for anything else it would become weird to name the variable itemPrice.